config/routes/claims.rb

```diff
@@ -1,7 +1,7 @@
 scope module: :claims, as: :claims, constraints: { host: ENV["CLAIMS_HOST"] } do
-  root to: "pages#index"
+  root to: "schools#show"
```

Comment:
Please leave this as `pages#index`. We will update the `after_sign_in_path` in another PR.